include/Data/Either.spec

```diff
@@ -1,5 +1,5 @@
 module spec Data.Either where
 
 measure isLeft :: Data.Either.Either a b -> Bool
```

Comment:
These are especially important as its new syntax -- I will need to update various bits of documentation appropriately!

Reply:
Indeed -- I guess we should state straight away that the parser is now layout-sensitive and therefore users are expected to format their specs appropriately, as they would do with "normal" Haskell programs.

Reply:
`measure`s are probably the most controversial aspect of the new parser. One could argue that a Haskell function definition is not indented wrt its type signature ...

So for measures, you can currently (with the new parser) do either
```
measure
  isLeft :: Data.Either.Either a b -> Bool
  isLeft ... = ...
  isLeft ... = ...
```
or
```
measure isLeft :: Data.Either.Either a b -> Bool
  isLeft ... = ...
  isLeft ... = ...
```

It is perhaps possible to even allow
```
measure isLeft :: Data.Either.Either a b -> Bool
isLeft ... = ...
isLeft ... = ...
```
but it requires a bit of additional work, and I am not sure whether it's worth it. If you think it is, I can look into it.